components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -142,96 +211,146 @@ service NodesService {
+	// Boolean to denote whether or not ssl should be checked when accessing the node.
 	bool ssl = 27;
+	// Boolean to denote whether or not self signed certificate should be allowed when accessing the node.
 	bool self_signed = 28;
+	// Username used to access the node (taken from credential id associated with node).
 	string user = 29;
-	repeated string key_files = 31;
+	// Sudo options to use when accessing the node.
 	string sudo_options = 33;
-	string region = 34;
-	string subscription_id = 35;
+	// List of hostnames (fqdn or ip address) for nodes to be created with bulk create.
 	repeated string hosts = 40;
 }
 
+// Node information.
 message Node {
+	// UUID for the node.
 	string id = 1;
+	// User-specified name for the node.
 	string name = 2;
+	// Platform for the node.
 	string platform = 3;
+	// Platform version for the node.
 	string platform_version = 4;
+	// Manager associated with the node (automate, aws-ec2, aws-api, azure-vm, azure-api, gcp).
 	string manager = 5;
+	// Tags to be applied to the node.
 	repeated chef.automate.domain.compliance.api.common.Kv tags = 20;
+	// End time from the most recent `detect` or `exec` job.
 	google.protobuf.Timestamp last_contact = 21;
+	// Status of the node (unreachable, reachable, unknown).
 	string status = 22;
+	// Details from the most recent scan job that executed for the node.
 	ResultsRow last_job = 23;
+	// Details for ssh/winrm access of the node.
 	TargetConfig target_config = 99;
+	// List of manager ids associated with the node.
 	repeated string manager_ids = 24;
+	// Most recent connection error received from attempting to contact the node.
 	string connection_error = 25;
+	// Last known state of the node (running, stopped, terminated).
 	string state = 26;
+	// Name prefix to attach to the node. The full node name is constructed based off the prefix and the host.
 	string name_prefix = 27;
+	// List of projects associated with the node. Projects are a concept introduced in IAMv2.
 	repeated string projects = 28;
+	// Most recent run data for the node, taken from the most recent infra run for the node. 
 	LastContactData run_data = 29;
+	// Most recent scan data for the node, taken from the most recent InSpec scan executed on the node. 
 	LastContactData scan_data = 30;
 }
 
+// Most recent run or scan data for the node, taken from the most recent Infra run or InSpec scan executed on the node. 
 message LastContactData {
-	string id = 1; // run id or report id
+	// Infra run report id or InSpec scan report id.
+	string id = 1;
 	enum Status {
 		UNKNOWN = 0;
 		PASSED = 1;
 		FAILED = 2;
 		SKIPPED = 3;
 	}
+	// Status on the last report for the node.
 	Status status = 2;
+	// Status on the next-to-last report for the node.
 	Status penultimate_status = 3;
+	// Endtime on the most recent report for the node.
 	google.protobuf.Timestamp end_time = 4;
 }
 
 message Nodes {
+	// List of node objects.
 	repeated Node nodes = 1;
+	// Total count of nodes in the system.
```

Comment:
```suggestion
	// Total number of nodes in the system.
```
Or `Count of nodes in the system.` Total count is redundant unless the count is incomplete.